pkgs/c_compiler/test/c_compiler_test.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Consider adding the Dart files ones they do something useful :-)

Reply:
I wanted to ensure we exercise the CI for tests and coverage.

Actual content is in progress =)